.github/workflows/build_appimage.sh

```diff
@@ -484,18 +484,26 @@ EOF
     libxkbcommon.so.0
   )
 
-  # fix AppImage output file name
+  # fix AppImage output file name, maybe not needed anymore since appimagetool lets you set output file name?
   sed -i 's/Name=qBittorrent.*/Name=qBittorrent-Enhanced-Edition/;/SingleMainWindow/d' /tmp/qbee/AppDir/usr/share/applications/*.desktop
 
-  APPIMAGE_EXTRACT_AND_RUN=1 \
-    /tmp/linuxdeployqt-continuous-x86_64.AppImage \
+  export APPIMAGE_EXTRACT_AND_RUN=1
+  /tmp/linuxdeployqt-continuous-x86_64.AppImage \
     /tmp/qbee/AppDir/usr/share/applications/*.desktop \
     -always-overwrite \
-    -appimage \
+    -bundle-non-qt-libs \
     -no-copy-copyright-files \
-    -updateinformation="zsync|https://github.com/${GITHUB_REPOSITORY}/releases/latest/download/qBittorrent-Enhanced-Edition-x86_64.AppImage.zsync" \
     -extra-plugins="$(join_by ',' "${extra_plugins[@]}")" \
     -exclude-libs="$(join_by ',' "${exclude_libs[@]}")"
+
+  # Workaround to use the static runtime with the appimage
+  export ARCH=x86_64
+  appimagetool_download_url="https://github.com/AppImage/appimagetool/releases/download/continuous/appimagetool-${ARCH}.AppImage"
+  [ -x "/tmp/appimagetool-${ARCH}.AppImage" ] || retry curl -kSLC- -o /tmp/appimagetool-"${ARCH}".AppImage "${appimagetool_download_url}"
+  chmod -v +x "/tmp/appimagetool-${ARCH}.AppImage"
+  /tmp/appimagetool-"${ARCH}".AppImage --comp zstd --mksquashfs-opt -Xcompression-level --mksquashfs-opt 20 \
+    -u "zsync|https://github.com/${GITHUB_REPOSITORY}/releases/latest/download/qBittorrent-Enhanced-Edition-$ARCH.AppImage.zsync" \
```

Comment:
`${ARCH}`, add `{}` to keep the coding style consistent.

Reply:
@abcfy2 I [tested adding the](https://github.com/Samueru-sama/qBittorrent-Enhanced-Edition/commit/d07195753d91c79433d029d38e04801498536a90) `{}` and that broke zsync.

```
/tmp/samuel/Volatile/qBittorrent-Enhanced-Edition-x86_64.AppImage/ appimageupdatetool ./qBittorrent-Enhanced-Edition-x86_64.AppImage -d
terminate called after throwing an instance of 'appimage::update::AppImageError'
  what():  Unknown AppImage type or not an AppImage
zsh: IOT instruction  appimageupdatetool ./qBittorrent-Enhanced-Edition-x86_64.AppImage -d
/tmp/samuel/Volatile/qBittorrent-Enhanced-Edition-x86_64.AppImage/                                               
```

[Artifact link](https://github.com/Samueru-sama/qBittorrent-Enhanced-Edition/actions/runs/10399719778)

This is how it should look instead, here is the output from the previous build: 

```
/tmp/samuel/Volatile/qBittorrent-Enhanced-Edition-x86_64.AppImage/ appimageupdatetool ./qBittorrent-Enhanced-Edition-x86_64.AppImage -d 
Parsing file: ./qBittorrent-Enhanced-Edition-x86_64.AppImage
AppImage type: 2
Raw update information: zsync|https://github.com/Samueru-sama/qBittorrent-Enhanced-Edition/releases/latest/download/qBittorrent-Enhanced-Edition-x86_64.AppImage.zsync
Update information type: Generic ZSync URL
Assembled ZSync URL: https://github.com/Samueru-sama/qBittorrent-Enhanced-Edition/releases/latest/download/qBittorrent-Enhanced-Edition-x86_64.AppImage.zsync
/tmp/samuel/Volatile/qBittorrent-Enhanced-Edition-x86_64.AppImage/
```

Reply:
try to add `export` before `ARCH="$(arch)"` ?

Reply:
[Same error still](https://github.com/Samueru-sama/qBittorrent-Enhanced-Edition/actions/runs/10402980706)

I'm rebuilding without any of the changes to see if the issue is something upstream.

Reply:
The error is still there, something broke upstream, very likely zsync.

I have a fork of the Brave appimage using this very appimagetool that produces nightly builds several times per day, the issue started about 24 hours ago.

Reply:
Alright, the issue was upstream and [was fixed](https://github.com/AppImage/type2-runtime/pull/70)

Reply:
> Alright, the issue was upstream and [was fixed](https://github.com/AppImage/type2-runtime/pull/70)

Great. So the PR is working as expect right ? Then I have no more to add. Maybe you can pin @c0re100 to merge your PR.

Reply:
Yes it is working as expected. Thanks for the review.

What a terrible luck I had that the appimage runtime broke midway during testing lol.